### PR TITLE
fix: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password

### PR DESCRIPTION
**Pull request by liyigang1**
When checking file information, if the host is down and can be accessed, a prompt box will pop up

Log: The mounted Samba network disk prompts that the file has been moved or deleted after the user changes their password
Bug: https://pms.uniontech.com/bug-view-261605.html

### CODE REVIEW

**Review by max-lvs — DISMISSED** [review]

File: src/plugins/filemanager/core/dfmplugin-workspace/views/fileview.cpp [hunk 1]
```diff
@@ -1915,6 +1917,18 @@ void FileView::initializeConnect()
         },
                 Qt::DirectConnection);
     }
+    connect(&FileInfoHelper::instance(), &FileInfoHelper::smbSeverMayModifyPassword, this, [this](const QUrl &url){
+        if (DeviceUtils::isSamba(rootUrl()) && url.path().startsWith(rootUrl().path())) {
+            qInfo() << rootUrl() << url << "smb server may modify password";
```
Comment (Johnson-zs) [inline src/plugins/filemanager/core/dfmplugin-workspace/views/fileview.cpp]: fmInfo

File: src/dfm-base/file/local/asyncfileinfo.h [hunk 2]
```diff
@@ -65,6 +65,7 @@ class AsyncFileInfo : public FileInfo
     // less 0,cache fialed, equeal 0,another cache, bigger 0 cache success
     int cacheAsyncAttributes();
     bool asyncQueryDfmFileInfo(int ioPriority = 0, initQuerierAsyncCallback func = nullptr, void *userData = nullptr);
+    DFMIOError errorDfm();
```
Comment (Johnson-zs) [inline src/dfm-base/file/local/asyncfileinfo.h]: 这个函数作为 const 函数好一些，其次名字很奇怪，叫 errorFromDFMIO ？

**Review by Johnson-zs — CHANGES_REQUESTED** [review]

File: src/plugins/filemanager/core/dfmplugin-workspace/views/fileview.cpp (hunk 1 above)
Comment (liyigang1) [reply src/plugins/filemanager/core/dfmplugin-workspace/views/fileview.cpp]: 已修改

File: src/dfm-base/file/local/asyncfileinfo.h (hunk 2 above)
Comment (liyigang1) [reply src/dfm-base/file/local/asyncfileinfo.h]: 已修改

**Review by Johnson-zs — APPROVED** [review]

**Pull request merged**